src/mudlet.cpp

```diff
@@ -506,14 +506,12 @@ QSettings* mudlet::getQSettings()
 
 void mudlet::initEdbee()
 {
-    // We only need the single Lua lexer, probably ever
-    // Optional additional themes will be added in future
-
-    edbee::Edbee* edbee = edbee::Edbee::instance();
-    edbee->autoInit();
+    auto edbee = edbee::Edbee::instance();
+    edbee->init();
     edbee->autoShutDownOnAppExit();
 
     auto grammarManager = edbee->grammarManager();
+    // We only need the single Lua lexer, probably ever
```

Comment:
🤔 Just a thought but there are two instances where we might/are using the edbee widget and it may not be displaying lua:
* When displaying the contents of a _variable_ - when that item type is selected in the editor - this _is_ using the existing edbee widget and may be okay as it is.  I do not know though whether we ever want to enable it to display other lua data types which are not currently handled in that view:
  * I think I have heard "**functions**" mentioned but I'm not sure those are doable - especially if they are represented as binary and not text {but then I may be conflating lua with python!)
  * The other main one would be **tables** - we might consider whether we could provide a "display(...)" type, read-only output for all the nodes beneath the table element (or even the whole table if the root node is selected in the `treeWidget_variables` {if that is its name}) - I dunno whether a custom lexer would be nice/wanted if we did _that_...
* We might wish to have a separate edbee-widget for the __button__/__menu__/__toolbar__ _CSS_ entry widget - and that would need a different lexer for the Qt specific pseudo-CSS language....